### ISSUE TITLE
history-graph to match X-axis for sensors with & w/out units

### ISSUE THREAD

**Issue by Masterz69** [home-assistant/frontend#10234]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

![image](https://user-images.githubusercontent.com/15225823/136783334-07b8d1d6-0731-4eb4-afe9-238f1447d909.png)

As seen on picture X-Axis not same places for different sensors inside same card. It make harder to match events with values.

### Describe the behavior you expected

Match X-axis placement - see red arrow.

### Steps to reproduce the issue

Just create history-graph card with entities with & w/out units.

### What version of Home Assistant Core has the issue?

core-2021.10.2

### In which browser are you experiencing the issue with?

93.0.4577.82 (Official Build) (64-bit) (cohort: 93_Win_82)

### Which operating system are you using to run this browser?

Windows 10 OS Version 2009 (Build 19042.1237)

### Problem-relevant frontend configuration

```yaml
- type: history-graph
            entities:
              - entity: switch.hw_pump_plg
                name: Hot Water Pump
              - entity: sensor.boiler_sns_h_hw
                name: Hot Water
              - entity: sensor.boiler_sns_h_hw_ret
                name: Return
              - entity: sensor.boiler_sns_h_cw
                name: Cold Water
            hours_to_show: 9
            refresh_interval: 60
```

**Comment by nukefrenzy** [comment]
I created this feature request on the forum, but I agree that this is more of a bug.  See here for my similar request, description of what I experience as well, and others that agree with this issue:

[https://community.home-assistant.io/t/history-make-all-times-line-up-on-the-history-page-for-easy-comparison/350322](url)

**Comment by berkavil** [comment]
seems like nobody cares about matching the X axis together... what a shame. :-( I would like to have in one graph historical statuses (string variables) - for heatpump, and in linear graph above the temperatures of heating water... so far it seems to be not possible to combine adequatelly.
There is a lot of workarounds and other cards to be used - e.g. the  apex chart graph, custom minigraph or finally graphana... all are superb in certain usage, but so far I was not successful to find this option to combine history statuses (string vars) with linear graphs (numerical values) on the same X-axis to see how they were relevant together in past few days... 
here are my trials (so others can save the time trying)
1) apex charts - great in zooming, spaning, dynamic focus time on X-axix, dynamic entities included => but no chance to show non-numeric values (string statuses of heatpump - like on/off//waiting/heating/blocked... etc.)
2) custom minigraph - great in simplicity, dynamic X-axis, dynamic entities, but no axes described, no time span (I didnt find the possibility to show 24 hours detailed history 5 days ago, only hours-to-show whats is great but not perfect :-)
3) history graph - great in showing string variables, but many (let's call it) bugs (even if it is not a bug - it is behavior), e.g. the colors are dynamic and can't be set easily for predefined statuses (yes there is a workaround with styles, but still  guite complicated for me - and I did not succeed yet), every time the HA page reloads, the colors are changing quite randomly. The X-axis is always moved a side a bit, since there are labels on left(or right) side and then the x-axis doesn't match the x-axis of linear graphs above) also the span is not possible, there is hours-to-show only.

the closest as I get with the resuls is combining custom minigraph and history (without span option, and if there is only one string variable the labels are hidden, so the two graphs quite match together...) but I guess it is still not the best what I was looking for.
thx.

**Comment by ildar170975** [comment]
Check https://github.com/home-assistant/frontend/issues/12839

**Comment by Masterz69** [comment]
Looks fixed in 2022.6.1.
It looks much better now.

 Thank You very much.
